packages/cashier/src/Components/Form/account-transfer-form.jsx

```diff
@@ -490,12 +495,12 @@ const AccountTransferForm = ({
                                         />
                                     </div>
                                 )}
-                                <div className='cashier__form-submit cashier__form-submit--align-end account-transfer__form-submit'>
+                                <div className='cashier__form-submit account-transfer-form__form-submit'>
                                     <Button
                                         className={classNames({
                                             'cashier__form-submit-button':
```

Comment:
Does `cashier` class also need to be imported or are these classes available in `account-transfer-form.scss`?

Reply:
Yes, `cashier` needs to be imported. This is one of the shared classes defined which are being used by different components.